src/screens/CreatePost/index.js

```diff
@@ -704,6 +704,15 @@ const CreatePost = () => {
     }
   }, [typeUser]);
 
+  React.useEffect(() => {
+    const followType = params?.followType;
+    if (followType === 'incognito' || followType === undefined) {
+      setTypeUser(true);
+    } else {
+      setTypeUser(false);
+    }
+  }, [params.followType]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useEffect` hook is used to set the `typeUser` state based on the `followType` parameter. However, it's not clear what `typeUser` represents in this context. Consider renaming `typeUser` to something more descriptive, like `isIncognito`, for better readability and understanding of its purpose.
```diff
-  const [typeUser, setTypeUser] = useState(false);
+  const [isIncognito, setIsIncognito] = useState(false);

-  React.useEffect(() => {
+  useEffect(() => {
     const followType = params?.followType;
     if (followType === 'incognito' || followType === undefined) {
-      setTypeUser(true);
+      setIsIncognito(true);
     } else {
-      setTypeUser(false);
+      setIsIncognito(false);
     }
   }, [params.followType]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->